# Definition of ROS service LocStartLocalizing for sick localization.
#
# ROS service LocStartLocalizing starts localization
# by sending cola command ("sMN LocStartLocalizing").
#
# See Telegram-Listing-v1.1.0.241R.pdf for further details about 
# Cola telegrams and this command.

#
# Request (input)
#

---

#
# Response (output)
#

bool success # true: success response received from localization controller (localization started), false: service failed (timeout or error status from controller)

